build.sbt

```diff
@@ -530,7 +509,6 @@ lazy val rchain = (project in file("."))
     regex,
```

Comment:
BTW what is this `regex` module? It seems unused in our build. Shouldn't we remove it as well?

Reply:
I guess since no module is depending on it we can remove it indeed. I think it was intended to be used in rspace but then never was used.